test.sh

```diff
@@ -290,7 +290,9 @@ function test_rustc() {
 
     rust_toolchain=$(cat rust-toolchain | grep channel | sed 's/channel = "\(.*\)"/\1/')
 
-    git clone https://github.com/rust-lang/rust.git || true
+    if [ ! -d "rust" ]; then
```

Comment:
Is this condition necessary, since the following line uses `|| true`?

Reply:
This is a good point. Reverting this one.